tests/run-make/pdb-buildinfo-cl-cmd/rmake.rs

```diff
@@ -17,23 +17,9 @@ fn main() {
         .crate_type("bin")
         .metadata("dc9ef878b0a48666")
         .run();
-    let tests = [
-        &env_var("RUSTC"),
-        r#""main.rs""#,
-        r#""-g""#,
-        r#""--crate-name""#,
-        r#""my_crate_name""#,
-        r#""--crate-type""#,
-        r#""bin""#,
-        r#""-Cmetadata=dc9ef878b0a48666""#,
-    ];
-    for test in tests {
-        assert_pdb_contains(test);
-    }
-}
 
-fn assert_pdb_contains(needle: &str) {
-    let needle = needle.as_bytes();
-    use bstr::ByteSlice;
-    assert!(&rfs::read("my_crate_name.pdb").find(needle).is_some());
+    let pdbutil_result =
+        llvm::LlvmPdbutil::new().arg("dump").arg("-ids").input("my_crate_name.pdb").run();
```

Comment:
Suggestion: can you add a simple helper like for the other llvm bin utils that just do

```rs
pub fn llvm_pdbutil() -> LlvmPdbutil {
    LlvmPdbutil::new()
}
```

for consistency?

Reply:
Sure. Will add this in the other pr (https://github.com/rust-lang/rust/pull/130155) and then fix the pr that did not get merged first.

Reply:
Hmm. Maybe its easier if I just close https://github.com/rust-lang/rust/pull/130155 and add its changes to this pr. Then we don't have do deal with merge problems.
Is this fine for you?

Reply:
Yes that's fine if you want to collapse that PR into this one.